File: src/main/java/io/github/kahenteikou/kotlinoder/App/MainApp.java
package io.github.kahenteikou.kotlinoder.App;

import javafx.application.Application;
import javafx.fxml.FXML;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

public class MainApp extends Application {
    @Override
    public void start(Stage stage) throws Exception {
        Parent root= FXMLLoader.load(getClass().getResource("../assets/MainWindow.fxml"));
        Scene scene=new Scene(root);
        stage.setTitle("MainWindow");
        stage.setScene(scene);
        stage.show();
    }
}

File: src/main/java/io/github/kahenteikou/kotlinoder/App/MainApp.kt
package io.github.kahenteikou.kotlinoder.App

import io.github.kahenteikou.kotlinoder.Controllers.MainWindowController
import io.github.kahenteikou.kotlinoder.assets.AssetStub
import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Node
import javafx.scene.Scene
import javafx.scene.layout.StackPane
import javafx.stage.Stage

class MainApp : Application() {
    lateinit private var controller : MainWindowController
    @Throws(Exception::class)
    override fun start(stage:Stage) {
        var l2:AssetStub = AssetStub()
        var loader: FXMLLoader = FXMLLoader(l2.javaClass.getResource("MainWindow.fxml"))
        var canvas:StackPane = StackPane()
        var scene : Scene= Scene(canvas, 800.0, 600.0)
        try{
            loader.load<Any>()
        }catch (e:Exception){
            e.printStackTrace()
        }
        controller = loader.getController<MainWindowController>()
        canvas.children.add((loader.getRoot() as Node))
        stage.title = "MainWindow"
        stage.scene=scene
        stage.show()

    }
}